test/parachain/src/service.rs

```diff
@@ -15,17 +15,12 @@
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
 use std::sync::Arc;
-
 use sc_executor::native_executor_instance;
 use sc_service::{AbstractService, Configuration};
 use sc_finality_grandpa::{FinalityProofProvider as GrandpaFinalityProofProvider, StorageAndProofProvider};
-
 use polkadot_primitives::parachain::CollatorPair;
-
 use cumulus_collator::CollatorBuilder;
-
-use futures::FutureExt;
-
+use futures::prelude::*;
```

Comment:
I'm no huge fan of blindly including preludes. (But it is okay :P )

Reply:
:sloth: will fix it :)

Reply:
done